system/ee/ExpressionEngine/Addons/relationship/ft.relationship.php

```diff
@@ -80,17 +80,6 @@ public function validate_settings($data)
 
         $validator = ee('Validation')->make($rules);
 
-        $validator->defineRule('gtLimit', function ($key, $value, $parameters, $rule) use ($data) {
```

Comment:
@intoeetive Did we just drop this validation as a whole?  Is it still needed?

Reply:
Yes. Please see @ops-andy comment: https://github.com/ExpressionEngine/ExpressionEngine/pull/1722#discussion_r786040361